python/lsst/sims/featureScheduler/basis_functions.py

```diff
@@ -895,3 +1156,313 @@ def __call__(self, indx=None):
         result[clouded] = hp.UNSEEN
 
         return result
+
+
+class Moon_avoidance_basis_function(Base_basis_function):
+    """Mark regions that are closer than a certain .
+
+    """
+    def __init__(self, nside=default_nside, condition_features=None, survey_features=None,
+                 moon_distance=30.):
+        """
+        Parameters
+        moon_distance: float (30.)
+            Minimum allowed moon distance. (degrees)
+        """
+        if nside is None:
+            nside = utils.set_default_nside()
+
+        self.nside = nside
+        if survey_features is None:
+            self.survey_features = {}
+        if condition_features is None:
+            self.condition_features = dict()
```

Comment:
Here the empty `conditions_features` dict is defined using `dict()` and in other basis functions it is defined by 
`self.condition_features = {}`.  I think it would be worth it to be consistent across the basis functions, unless there is some reason to use `dict()` over `{}`. It also seems that `{}` is computationally faster than `dict()`, but that is only based on some very quick research.